edits to the blog (#90)
kuzudb/kuzu-blog: src/content/post/2025-02-03-kuzu-v-0.8.0.md

## Closing remarks

> We extend our thank to the entire Kùzu team and our incredible

Reviewer: ```suggestion
We extend our thanks to the entire Kùzu team and our incredible
```

interns who have worked very hard to make this release possible. We are thrilled to

> see Kùzu becoming a more and more feature-rich databse and encourage all of you try out these new features!

Reviewer: ```suggestion
see Kùzu becoming a more and more feature-rich database and encourage all of you try out these new features!
```

As usual please share your experiences with us – your feedback continues to
shape Kùzu's evolution. From the sometimes unreasonable cold of 🇨🇦, where it was -23 °C last week,

> enjoy the new release! 🎉 🎉

Reviewer: ```suggestion
we hope you enjoy the new release! 🎉 🎉
```

Till next time!

> ps: Oh, and no tariffs to 🇺🇸; it's free and open-sourced!

Reviewer: ```suggestion
PS: Oh, and we promise, no tariffs to 🇺🇸; Kùzu is free and open-source!
```
